[2022-07-22 13:00:18,259] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-07-22 13:00:18,485] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-07-22 13:00:18,492] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:18,494] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:18,496] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:18,645] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-06-26 00:00:00+00:00
[2022-07-22 13:00:18,673] {standard_task_runner.py:52} INFO - Started process 803 to run task
[2022-07-22 13:00:18,711] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpjzsk4p6j', '--error-file', '/tmp/tmp476k1p8h']
[2022-07-22 13:00:18,738] {standard_task_runner.py:80} INFO - Job 47: Subtask t1
[2022-07-22 13:00:18,875] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:19,172] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:19,713] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-07-22 13:00:19,722] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-22 13:00:19,926] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220626T000000, start_date=20220722T130018, end_date=20220722T130019
[2022-07-22 13:00:20,038] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:20,225] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:03,532] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-07-25 09:18:03,579] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-07-25 09:18:03,581] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:03,586] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:03,590] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:03,677] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-06-26 00:00:00+00:00
[2022-07-25 09:18:03,689] {standard_task_runner.py:52} INFO - Started process 765 to run task
[2022-07-25 09:18:03,713] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpov1gnmr7', '--error-file', '/tmp/tmpxoora7ra']
[2022-07-25 09:18:03,742] {standard_task_runner.py:80} INFO - Job 48: Subtask t1
[2022-07-25 09:18:03,872] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:04,062] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-06-26T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:04,372] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-07-25 09:18:04,375] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-25 09:18:04,512] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220626T000000, start_date=20220725T091803, end_date=20220725T091804
[2022-07-25 09:18:04,579] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:04,669] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
